### PR TITLE
A go at Starting fresh

### PR DESCRIPTION
**Pull request by RubenOlsen**
We now have quite a few changes - not only moving the *generic envelope budget* stuff out.

In short:
* Added section on handling cash. The idea behind this is to give people that are new to this kind of budgeting, an easy idea on what to do with cash. Even seasoned Zero-Summers can forget about this simple trick :-)
* Amended Setting up your Budget Categories with a small tip.
* Added sub section Group Your Expense Categories. I have guided a lot of people on using Zero-Sum software (Ynab, Financier - and now Actual) - and the grouping I propose have shown to help people to not touch category money they should not touch.
* Assigning Available Funds is a complete rewrite - this is "Tips" we removed from the envelope-budget.md file - I will add a link back into the getting-started document when both PRs has been done.
* Added a section on finding your budget number if you do not want, or can use historical data
* Keep Budgeting was amended 
* Added a header before text on next step.

OBSERVATIONS: I think this document is too big - and it should be split into it's 5 parts (+ introduction).

### CODE REVIEW

File: docs/getting-started/starting-fresh.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+
+![](/img/getting-started/actual-intro-set-budget-to-3-months-average.png)
+
+You can also set specific categories to various averages::
+
+![](/img/getting-started/actual-intro-set-budget-category-average.png)
+
+
+### What to do if you do not want to, or can, use historical data
```
Comment (youngcw) [inline docs/getting-started/starting-fresh.md:224]: "or can't"  ??

**Comment by youngcw** [comment]
I feel like breaking it up would make for really small articles.  One looks like it would be 2 sentances.

**Comment by RubenOlsen** [comment]
> I feel like breaking it up would make for really small articles. One looks like it would be 2 sentences.

The last one would have to go into the last file of course. But it's up to you guys. Research into UX on text shows that unless you are accustomed to reading long text, you will loose consentration. Personally I have no strong feeling for either-or - it was just a suggestion from my end :)


Talking about small articles - my next "project" is to take on the "Taking a Tour of Actual" - there we have a lot "small" articles :-D

**Comment by youngcw** [comment]
> The last one would have to go into the last file of course. But it's up to you guys. Research into UX on text shows that unless you are accustomed to reading long text, you will loose consentration.

I get that.  I think in this case, at least for me.  I think I would appreciate this being or check list like.  So I did that now I do this, and I can see the list and not have to flip back and forth

File: docs/getting-started/starting-fresh.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+* **Bills (Flexible)**: All bills that vary month to month
+* **Savings**: Funds you have saved, or are going to transfer to an off budget savings account
+* **General**: Everything else
+
+Some other common categories you could add would be rent/mortgage, taxes, eating out, specific utilities, 
+subscription services, charitable donations, child care, gifts, fun money, or debt payment.
+
+:::tip
+
+Refrain from overthinking what categories you want from the start, your situation is
```
[the remark this answers is not in the document]
Comment (RubenOlsen) [reply docs/getting-started/starting-fresh.md:86]: overthinking is a word in the dictionary.

**Review by youngcw — APPROVED** [review]
Ill merge this in for now and it can be edited or split up later

**Pull request merged**